### DISCUSSION TITLE
Would be amazing if you create a toolbox for Nikon's .NEV files

### DISCUSSION THREAD

**Discussion by helidm** [latenitefilms/BRAWToolbox#129] [Feature Requests]
FCPX doesn't support the new RAW format from Nikon (.nev).  I think you'd make a killing by adapting  BRAW toolbox to that codec.  It is a version of TicoRaw.   I hope this could be a fun project for you.  Thanks!

**Comment by latenitefilms** [comment]
Thanks for the suggestion @helidm! 

I've reached out to intoPIX to see if we can get access to the TicoRAW SDK/API.

**Reply by helidm** [reply]
Amazing!  This would be a very timely application as new Nikon Z8 cameras hit the market with no support on FCPX.

**Reply by latenitefilms** [reply]
I'm in contact with Nikon, so we'll see what happens.

**Reply by latenitefilms** [reply]
For anyone following along, I'm actively exploring this idea. No ETA or promise, but will let you know if it ends up happening. Stay tuned.

**Reply by helidm** [reply]
Looks like a year later Color Finale has designed a solution.  They are selling it for $139

**Reply by latenitefilms** [reply]
Yes, since Color Finale has already got a good working solution, I've stopped development of my own Nikon implementation.
